src/magicbox/SuggestionsManager.ts

```diff
@@ -1,8 +1,9 @@
-import { $$ } from '../utils/Dom';
+import { $$, Dom } from '../utils/Dom';
 import { InputManager } from './InputManager';
 import { each, defaults, indexOf, compact } from 'underscore';
-import { OmniboxEvents } from '../Core';
+import { OmniboxEvents, Component } from '../Core';
 import { IQuerySuggestSelection } from '../events/OmniboxEvents';
+import { QuerySuggestPreview } from '../ui/QuerySuggestPreview/QuerySuggestPreview';
```

Comment:
The problem with this is that you will always be importing this component, and as such, bloating the resulting omnibox files with the complete code implementation of that component. That's not super nice for load times, or in lazy loading mode.

A way around that would be to import only the TypeScript types ( https://basarat.gitbooks.io/typescript/docs/project/external-modules.html , look for type imports ), or to go through the search interface reference and do `searchInterface.getComponents("QuerySuggestPreview")`